sumatra/commands.py

```diff
@@ -126,7 +126,7 @@ def init(argv):
                             description=description)
     parser.add_argument('project_name', metavar='NAME', help="a short name for the project; should not contain spaces.")
     parser.add_argument('-d', '--datapath', metavar='PATH', default='./Data', help="set the path to the directory in which smt will search for output datafiles generated by the simulation/analysis. Defaults to %(default)s.")
-    parser.add_argument('-i', '--input', metavar='PATH', default='/', help="set the path to the directory relative to which input datafile paths will be given. Defaults to the filesystem root.")
+    parser.add_argument('-i', '--input', metavar='PATH', default='.', help="set the path to the directory relative to which input datafile paths will be given. Defaults to the filesystem root.")
```

Comment:
I had also thought about this and I like the idea that the Sumatra project folder should be complete enclosed. If you want to implement input data, you can create a symbolic link to the Sumatra project folder.

Furthermore, for the project file I use relative path for datastores. It would look ugly like '../../../../', if we use '/' as default root for input data store. After moving the folder to other path, we have to change the default root for input data store.

@other: What do you think?